cypress/integration/seededFlows/profileFlows/followOrganisation.spec.js

```diff
@@ -12,24 +12,21 @@ describe('Follow user from profile page', () => {
   it('follows and unfollows an organisation', () => {
     cy.intercept('/follows').as('followsRequest');
 
-    cy.findByRole('button', {
-      name: 'Follow organization: Bachmanity',
-    }).click();
-    cy.wait('@followsRequest');
-    cy.findByRole('button', { name: 'Unfollow organization: Bachmanity' });
+    cy.findByRole('button', { name: 'Follow organization: Bachmanity' }).as(
+      'followButton',
+    );
 
-    // Check that the update persists after reload
-    cy.visitAndWaitForUserSideEffects('/bachmanity');
-
-    cy.findByRole('button', {
-      name: 'Unfollow organization: Bachmanity',
```

Comment:
like above, it would be good to keep this check in but change it to e.g. `cy.findByRole('button', { name: 'Follow organization: Bachmanity' }).should('have.attr', 'aria-pressed', 'true')` to check we're still in the 'following' state